libs/ui/src/themes/render_with_themes.tsx

```diff
@@ -1,16 +1,48 @@
 import React from 'react';
-import { render, RenderOptions, RenderResult } from '@testing-library/react';
+import {
+  queries,
+  render,
+  RenderOptions,
+  RenderResult,
+} from '@testing-library/react';
 
 import { ColorMode, SizeMode } from '@votingworks/types';
 import { AppBase } from '../app_base';
 
-type VxRenderOptions = RenderOptions & {
+export type VxRenderOptions = RenderOptions & {
   vxTheme?: {
     colorMode?: ColorMode;
     sizeMode?: SizeMode;
   };
 };
 
+export interface ButtonQueryOptions {
+  useSparinglyIncludeHidden?: boolean;
+}
+
+export interface OptionQueryOptions {
+  expectSelected?: boolean;
+}
+
+export type VxRenderResult = RenderResult & {
+  getAllButtons(
+    buttonText: string | RegExp,
+    opts?: ButtonQueryOptions
+  ): ReturnType<queries.AllByRole>;
+  getButton(
+    buttonText: string | RegExp,
+    opts?: ButtonQueryOptions
+  ): ReturnType<queries.GetByRole>;
+  findAllButtons(
+    buttonText: string | RegExp,
+    opts?: ButtonQueryOptions
+  ): ReturnType<queries.FindAllByRole>;
+  findButton(
+    buttonText: string | RegExp,
+    opts?: ButtonQueryOptions
+  ): ReturnType<queries.FindByRole>;
+};
+
```

Comment:
This is great! Thanks for making this happen!

I would prefer, however, that we extend the `screen` option to include these queries rather than destructuring the render result in tests. Using `screen` is [recommended practice](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen) and I think just easier. 

It doesn't matter much to me if you change the tests you've updated to use the `screen` object - I think it's OK that we update the tests as we go, and I'm sure you've already done a lot of fishing in the all the frontends. But I like the idea of putting these queries on the `screen` in this PR or a follow-up PR. 

Thoughts?